core-bundle/src/Resources/views/Frontend/preview_toolbar_base.html.twig

```diff
@@ -3,14 +3,14 @@
     <a href="#" title="{{ 'MSC.openToolbar'|trans }}">{{ include('@ContaoCore/Collector/contao.svg') }}</a>
 </div>
 <div class="cto-toolbar__inside">
-    <form action="{{ action }}" method="post">
+    <form action="{{ action }}" method="post" autocomplete="off">
```

Comment:
Did you test that the datalist autosuggest still works this way?

Reply:
The auto suggest currently does not work in Firefox at all (#3733). But if I both add `autocomplete="off"` to the `<form>` _and_ remove `placeholder` from the input, it works.